apps/prairielearn/src/pages/instructorIssues/instructorIssues.sql

```diff
@@ -86,3 +124,47 @@ LIMIT
   $limit
 OFFSET
   $offset;
+
+-- BLOCK close_issues
+WITH
+  updated_issues AS (
+    UPDATE issues AS e
```

Comment:
Nit: why `e` instead of `i`?

Reply:
This is a remnant of the fact that this table was originally called `errors` back in 2017 😂 fixed in aef408e.